linera-service/tests/linera_net_tests.rs

```diff
@@ -2867,9 +2867,10 @@ async fn test_end_to_end_faucet_with_long_chains(config: impl LineraNetConfig) -
 
     // Use the faucet directly to initialize many chains
     for _ in 0..chain_count {
-        faucet_client
+        let (_, new_chain_id) = faucet_client
             .open_chain(faucet_chain, None, Amount::ONE)
```

Comment:
I think we wouldn't need all this if we just passed in a new random public key; then the faucet doesn't know the secret key from the beginning and doesn't try to follow it. Or is it important that it's added to the wallet first?

Reply:
That's a good idea! I tried to do it now, but just adding `Some(PublicKey::test_key(1))` caused the test to fail, so I opened #2723 as a follow-up issue.